src/locale/localed-util.c

```diff
@@ -136,6 +256,17 @@ int x11_read_data(Context *c, sd_bus_message *m) {
         struct stat st;
         int r;
 
+        assert(c);
+
+        r = vconsole_read_data(c, m);
+        if (r < 0)
+                return r;
+
+        if (!x11_context_isempty(&c->x11_from_vc)) {
+                log_debug("XKB settings loaded from vconsole.conf, not reading xorg.conf.d/00-keyboard.conf.");
+                return 0;
+        }
```

Comment:
@poettering I'm thinking if this is really desirable...

IMO we should respect more what is actually applied to X11, i.e `00-keyboard.conf` should have higher priority if that exists.

Reply:
And if the two get desynchronized, localed should be responsible for overwritting `vconsole.conf` based on `00-keyboard.conf`.